sdk/src/transaction/sanitized.rs

```diff
@@ -278,7 +293,7 @@ impl SanitizedTransaction {
     }
 
     /// Validate a transaction message against locked accounts
-    fn validate_account_locks(
+    pub fn validate_account_locks(
```

Comment:
if this is going to be a pub function of SanitizedTransaction, may as well make it `pub fn validate_account_locks(&self, tx_account_lock_limit: usize)`